public/scripts/common/lib/handlebars.mixed.js

```diff
@@ -29,10 +29,53 @@ Displays select tag with options (and optional optgroups)
   * `selected`
 
 **/
+
 Handlebars.registerHelper('select', function(context, options) {
-  var output = `<select id="${options.hash.id}" name="${options.hash.name}" class="${options.hash.class}">`;
-  output += makeOptions(context, options);
-  return output +'</select>';
+  var addOption, addOptions,
+      output = '';
+
+  addOption = function(selected, _options) {
```

Comment:
Hey @OkayDave could you refactor the `makeOptions` function https://github.com/Synbiota/GENtle2/blob/feature/require_rdp_part/public/scripts/common/lib/utils.js#L42-L67 to include the additional functionality you've added please?